src/stpipe/log.py

```diff
@@ -108,7 +108,7 @@ def __init__(self, name, handler=None, level=logging.NOTSET,
         self.name = name
         self.handler = handler
         if not isinstance(self.handler, list):
-            if self.handler.strip() == '':
+            if self.handler is None or self.handler.strip() == '':
```

Comment:
What does this fix?  I wonder if this needs to be backported to `jwst.stpipe`?

Reply:
The `handler` attribute is set to None in the tox environment.  I didn't take the time to figure out why, but since jwst doesn't run tests in tox it's not had this issue.